### PR TITLE
insert product

### PR DESCRIPTION
**Pull request by utkarsh-gupta-dt**
Created a script to add products

### CODE REVIEW

File: product/product.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO Products (ProductID, ProductName, Price, StockQuantity)
+VALUES
+    (4, 'Smart Watch', 149.99, 30);
```
Comment (yatharthmathur) [inline product/product.sql]: add a line at the end

**Review by yatharthmathur — CHANGES_REQUESTED** [review]
Address the issues

File: product/product.sql (hunk 1 above)
Comment (utkarsh-gupta-dt) [reply product/product.sql]: Fixed

**Review by yatharthmathur — APPROVED** [review]
LGTM

**Pull request merged**